tests/kafkatest/services/kafka/templates/kafka.properties

```diff
@@ -126,3 +126,11 @@ offsets.topic.num.partitions={{ num_nodes }}
 offsets.topic.replication.factor={{ 3 if num_nodes > 3 else num_nodes }}
 # Set to a low, but non-zero value to exercise this path without making tests much slower
 group.initial.rebalance.delay.ms=100
+
+group.coordinator.rebalance.protocols=classic,consumer,share
```

Comment:
There needs to be a better way to manage the configs here. Changing this file changes the configs for all of the tests, and that doesn't seem right when KIP-932 is still in early access. I think you should not include this change in the PR.